api/Navigator.json

```diff
@@ -2450,20 +2428,9 @@
             "chrome_android": {
               "version_added": "40"
             },
-            "edge": [
-              {
-                "version_added": "17"
-              },
-              {
-                "version_added": "16",
-                "flags": [
-                  {
-                    "type": "preference",
-                    "name": "Enable service workers"
```

Comment:
Can you do a PR that removes all of "Enable service workers" in one go? That way is more likely to reveal problems in the data.